stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/ChannelActivity.kt

```diff
@@ -95,10 +108,45 @@ class ChannelActivity : AppCompatActivity() {
                 currentUser = user
             )
 
-            ChannelList()
+            ChannelList(
+                itemContent = {
+                    CustomChannelListItem(channel = it, user = user)
+                }
+            )
         }
     }
 
+    /**
+     * An example of a customized DefaultChannelItem component.
+     */
+    @Composable
+    fun CustomChannelListItem(channel: Channel, user: User?) {
+        DefaultChannelItem(
+            channel = channel,
+            currentUser = user,
+            onChannelLongClick = { },
+            onChannelClick = { },
+            divider = {
+                Spacer(
+                    modifier = Modifier
+                        .fillMaxWidth(0.5f)
+                        .height(0.5.dp)
+                        .align(CenterHorizontally)
+                        .background(color = ChatTheme.colors.textLowEmphasis)
+                )
+            },
+            trailingContent = {
+                Spacer(modifier = Modifier.width(8.dp))
+            },
+            detailsContent = {
+                Text(
+                    text = it.getDisplayName(),
+                    style = ChatTheme.typography.bodyBold
+                )
```

Comment:
How about adding the `textColor`? We can't see the message on the dark mode. 😃 
```kotlin
color = ChatTheme.colors.textHighEmphasis
```

<img src="https://user-images.githubusercontent.com/24237865/139861008-591c0641-c836-47a7-9e5c-8077fa2dc337.jpeg" width="33%" />

Reply:
Good point!